docs/en/overview/Glossary.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: global
+title: Glossary
+group: Overview
+priority: 1
+---
+
+<!-- * Table of Contents
+{:toc} -->
+
+Glossary of Alluxio terminology
+
+<!-- what words should be included in glossary? best if can point to more detailed doc-->
```

Comment:
we can brainstorm more terms to add here later. you may end up adding more as you spend more time going through the docs content later